scripts/generate_settings.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+import yaml
+import jinja2
+import re
+from settings_list import SettingsList
+import os
+
+sbp_generate_path = "../libswiftnav/sbp_generate"
+myenviron = os.environ
+texinputs = myenviron["TEXINPUTS"]
+myenviron["TEXINPUTS"]=".:" + sbp_generate_path + ":" + texinputs
+
+settings = SettingsList("settings.yaml")
+groups = settings.sl_return_groups()
+
+LATEX_SUBS = (
+    (re.compile(r'\\'), r'\\textbackslash'),
+    (re.compile(r'([{}_#%&$])'), r'\\\1'),
+    (re.compile(r'~'), r'\~{}'),
+    #(re.compile(r'\^'), r'\^{}'),
```

Comment:
Yes I will remove, I'll replace with a comment indicating how these latex substitions differ from sbp_generate.py from where this script originated.